rb/lib/htmlproofer/enforce_img_https.rb

```diff
@@ -0,0 +1,18 @@
+class EnforceImgHTTPS < ::HTMLProofer::Check
+  def run
+    @html.css('img').each do |node|
+      @img = create_element(node)
+      line = node.line
+
+      if nonsecure?
+        return add_issue("Don't include images from an insecure location!", line: line)
```

Comment:
> This image src is insecure.
> Including it would result in marking the whole page as insecure through mixed-content.
?

(les erreurs de ce linter sont généralement contextualisées à la ligne ayant levé l'erreur, me semble-t-il)